src/cli.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+/* eslint no-console:0 */
+
+import minimist from 'minimist';
+import os from 'os';
+import path from 'path';
+import sh from 'shelljs';
+import tsts from 'tsts';
+
+const argv = minimist(process.argv.slice(2));
+
+// Get a full list of spells.
+const spells = sh.ls(path.join(__dirname, 'spells'))
```

Comment:
Also, would be cool for it to look for `videojs-spellbook` modules that are installed. Though, that's probably a spellbook 2.0 feature.
Depending on how complicated some of these spells end up being, we should split them up into separate modules themselves as well.